doc/user/content/self-hosted/_index.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "Materialize Kubernetes Operator"
+description: ""
+
+---
+
+## Materialize Kubernetes Operator
+
+The Materialize Kubernetes Operator deploys Materialize into a Kubernetes
+cluster.
+
+This guide provides instructions to install the Materialize Operator using Helm.
+
+## Prerequisites
+
+### Helm 3.2.0+
+
+If you don't have Helm version 3.2.0+ installed, refer to the [Helm
+documentation](https://helm.sh/docs/intro/install/).
+
+### Kubernetes
+
+Materialize supports [Kubernetes 1.19+](https://kubernetes.io/docs/setup/).
+
+### `kubectl`
+
+This tutorial uses `kubectl`. To install, refer to the [`kubectl` documentationq](https://kubernetes.io/docs/tasks/tools/).
+
+### Kubernetes Storage Configuration
```

Comment:
just noting that this is lacking context. we should have an explainer on why local storage is valuable (spilling to disk, operating on datasets larger than main memory, more graceful degradation rather than OOMing), and also note that this is optional (though highly recommended)